### DISCUSSION TITLE
[Idea] Allow repo sorting (by stars, date, etc.)

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#51716] [Other Feature Feedback, Questions, & Ideas]
Currently, lists are not sortable in any way whatsoever (not even manual click-dragging). This makes this feature lag behind other websites like YouTube, which allows sorting by:

- Date added (newest)
- Date added (oldest)
- Most popular
- Date published (newest)
- Date published (oldest)

If this functionality is not added, when lists get too long, re-ordering could require un-starring dozens of repo's just to move a single one!

**Comment by naji567** [comment]
M

**Reply by adamlui** [reply]
@naji567 what is m?

**Comment by ivanperez-keera** [comment]
Whenever we add sorting, can we always add alphabetical sorting please? That will help enormously especially when working with many repos.
